src/zenml/integrations/slack/steps/slack_alerter_ask_step.py

```diff
@@ -0,0 +1,53 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+"""Step that allows you to send messages to Slack and wait for a response."""
+
+from zenml.alerter.alerter_utils import get_active_alerter
+from zenml.integrations.slack.alerters.slack_alerter import (
+    SlackAlerter,
+    SlackAlerterConfig,
+)
+from zenml.steps import StepContext, step
+
+
+@step
+def slack_alerter_ask_step(
+    config: SlackAlerterConfig, context: StepContext, message: str
+) -> bool:
+    """Posts a message to the slack alerter component and waits for approval.
+
+    This can be useful, e.g. to easily get a human in the loop before
+    deploying models.
+
+    Args:
+        config: Runtime configuration for the slack alerter.
```

Comment:
```suggestion
    """Posts a message to the Slack alerter component and waits for approval.

    This can be useful, e.g. to easily get a human in the loop before
    deploying models.

    Args:
        config: Runtime configuration for the Slack alerter.
```